### ISSUE TITLE
How does the reg part influence the model weights?

### ISSUE THREAD

**Issue by pepsi2222** [BaiTheBest/SRDML#2]
the reg loss is a sum of distances, and a distance is the L1-norm of differences between two first-order deviations. But the tensorflow doesn't consider the second-order deviations when back propagating. It seems that the reg loss doesn't influence the model weights and only the cross entropy loss can influence the model weights.
